### PR TITLE
[WIP] Enable to connect to main category api server

### PR DESCRIPTION
**Pull request by smilenannan**
# Ticket/Issue url
(Summary of your pull request. Also link to JIRA tickets or Github issues if exists.)
https://elaly-inc.atlassian.net/browse/CLIENT-1

### screen shot
(Please upload screenshots or GIF animations. Required if the PR change GUI)

# Review Point
(Please indicate the points where the reviewer needs to check.)

# Estimated Impact Area
(Please share the estimated impact area if any.)

### Existing Function
(Describe the change to the existing function if any. Especially when refactored/followed the Boy Scout Rule.)

### Existing Operation
(Describe here if the change affects the current operation.)

## Security
(Personal information: Describe here if there is any concerns on personal information (e.g. email/address/name).)
(Authentication: Describe here if the cange affects access authorization.)

# QA List
(Link to the test case list or list up QA test cases in C1 coverage.)

# Checklist

- [ ] Have you filled all the items in the PR?
- [ ] Have you written tests? (minimum C1 coverage)
- [ ] Does the PR depend on any other repositories? If so, is it already described in the PR?
- [ ] Is there any prerequisites for this PR, such as batch files? If so, is it described properly in this PR?

### CODE REVIEW

File: src/repositories/FurnitureMainCategoryRepository.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import FurnitureMainCategory from '../entities/FurnitureMainCategory';
+
+class FurnitureMainCategoryRepository {
+  constructor(fetcher) {
+    this.fetcher = fetcher;
+  }
+
+  fetch() {
+    return this.fetcher.get('v1/main_categories')
+      .then(res => FurnitureMainCategory.fromJson(res.data));
```
[the remark this answers is not in the document]
Comment (smilenannan) [reply src/repositories/FurnitureMainCategoryRepository.js]: furnitureRepository.jsを真似て書いて見たのですが、動いてないっぽいので調査中です

File: src/actions/FurnitureMainCategory.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import * as FurnitureMainCategoryActionTypes from '../constants/FurnitureMainCategoryActionTypes';
```
[the remark this answers is not in the document]
Comment (smilenannan) [reply src/actions/FurnitureMainCategory.js]: このこのファイル名の頭文字も小文字にした方がいいですよね

[the remark this answers is not in the document]
Comment (MasanoriOyabu) [reply src/actions/FurnitureMainCategory.js]: そそー！いいね！わかってきたね！

[the remark this answers is not in the document]
Comment (smilenannan) [reply src/actions/FurnitureMainCategory.js]: スネークケースじゃなっくて、キャメルケース使ってるのって、それがreactライクだからって感じですかね(本質的な質問じゃない気がしますが)

[the remark this answers is not in the document]
Comment (MasanoriOyabu) [reply src/actions/FurnitureMainCategory.js]: react以前に、JavaScriptのスタンダード

File: src/actions/FurnitureMainCategory.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+export const fetchInitialFurnitureMainCategories = furniture_main_category => ({
```
[the remark this answers is not in the document]
Comment (smilenannan) [reply src/actions/FurnitureMainCategory.js]: ん、furniture_main_category はsnake caseですけど(掴めてない感)

**Pull request closed without merging**